content/examples/databricks-cluster.Rmd

```diff
@@ -0,0 +1,83 @@
+---
+title: "Using sparklyr with Databricks"
+output:
+  html_document:
+    toc: true
+    toc_depth: 2
+    toc_float:
+      collapsed: false
+params:
+  width: 600
+---
+
+## Overview
+
+This documentation demonstrates how to use `sparklyr` with RStudio Server Pro
+and a Databricks cluster with Apache Spark.
+
+## Best practices
+
+- **Connect to Spark remotely** - Install RStudio Server Pro outside of the
+  Databricks cluster and work with Spark remotely
+- **Work across RStudio Team** - Install RStudio Connect and RStudio Package
+  Manager outside of the Databricks cluster so that they are not limited to the
+  resources or ephemeral nature of Databricks clusters
+- **Restrict workloads to interactive analysis** - Only perform workloads
+  related to exploratory or interactive analysis with Spark, then write the
+  results to a database, file system, or cloud storage for more efficient
+  retrival in apps, reports, and APIs
+- **Load and query results efficiently** - Shiny apps that use Spark on the
+  backend tend to have performance and runtime issues; instead, consider reading
+  the results from a database, file system, or cloud storage instead
+- **Maintain a persisent installation** - If you cannot work with Spark
+  remotely, you should install RStudio Server Pro on the Driver node of a
+  long-running, persistent Databricks cluster as opposed to a worker node or an
+  ephemeral cluster
```

Comment:
These best practices read to me like an ordered list, so it feels like this is just the next step in that list when it is actually an alternative to steps 1-2. I think this would be better served as an alternative to step 1.

Reply:
Thanks for the feedback here. I tried a few things such an an ordered list and had intentionally dropped this item down to the end.

I reworked this in 1ca22c5, kept the unordered list, but moved the comment about "maintaining a persistent installation" to the bottom section for "Option 2 - Working inside of Databricks".